pennylane/drawer/tape_text.py

```diff
@@ -264,7 +326,21 @@ def tape_text(
     cache["tape_offset"] += len(tape_cache)
     for i, nested_tape in enumerate(tape_cache):
         label = f"\nTape:{i+current_tape_offset}"
-        tape_str = tape_text(nested_tape, wire_order, show_all_wires, decimals, max_length, cache)
+        tape_str = tape_text(
+            nested_tape,
+            wire_order,
+            show_all_wires,
+            decimals,
+            max_length,
+            show_matrices=False,
```

Comment:
Is there a way to show the matrices of nested tapes? Would we just have to expand the tapes or is it completely blocked because of this line?

Reply:
Nested tapes will all use the same cache dictionary, so the matrices will only be printed once at end. The print out at the end will be for all nested tapes. I can add a test for this though.

Reply:
Ohh I see, makes sense !